CMakeLists.txt

```diff
@@ -2,15 +2,22 @@ cmake_minimum_required(VERSION 3.12)
 
 project(CubicInterpolation LANGUAGES CXX)
 
+if (EXISTS ${CMAKE_BINARY_DIR}/conan_paths.cmake)
+    include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
+endif()
```

Comment:
It won't play very well with conan CCI recipes. I would recommend to completly remove conan stuff in CMakeLists, and call conan with `cmake_find_package` generator to generator Find files.